tests/test_policy.py

```diff
@@ -679,7 +679,8 @@ def test_valid_arn_type(self):
             'event-rule-target',
             'rrset',
             'redshift-reserved',
-            'elasticsearch-reserved'
+            'elasticsearch-reserved',
+            'receipt-rule-set'
```

Comment:
this should be `ses-receipt-rule-set`

Reply:
Agreed, on the community meeting I suggested removing the ARN altogether but looking with full eyes... your way lets this resource type generate synthetic ARNs which makes more sense :+1: